### DISCUSSION TITLE
scaling parameter of Transformer always set to True

### DISCUSSION THREAD

**Discussion by StanislasGuinel** [awslabs/gluonts#1907] [General]
The `scaling` parameter is always set to True when creating the `TransformerTrainingNetwork` or the `TransformerPredictionNetwork` but it is still a parameter of the `TransformerEstimator`.
Is there a reason why ?

**Comment by lostella** [comment]
Probably not. Without even looking into the details of how this option is used (and whether or not `scaling=False` makes sense), if the network exposes the option, and the estimator does too, it only makes sense to use whatever is set in the estimator.

Turning this into an issue, even though the fix should be trivial.
